arwn/engine.py

```diff
@@ -196,6 +196,10 @@ def on_message(client, userdata, msg):
             handlers.run(self, msg.topic, payload)
             return True
 
+        if config["mqtt"]["username"] and config["mqtt"]["password"]:
```

Comment:
this will cause a key error as is. I'll make the fix to do a get instead.
```suggestion
        if config["mqtt"].get("username") and config["mqtt"].get("password"):
```